libs/map/bookmark_helpers.cpp

```diff
@@ -503,6 +517,14 @@ std::vector<std::string> GetFilePathsToLoadFromKml(std::string const & filePath)
   return {std::move(fileSavePath)};
 }
 
+std::vector<std::string> GetFilePathsToLoadFromGeoJson(std::string const & filePath)
+{  // Copy input file to temp GeoJson with unique name.
+  auto fileSavePath = GenerateValidAndUniqueFilePathForGeoJson(base::FileNameFromFullPath(filePath));
+  if (!base::CopyFileX(filePath, fileSavePath))
+    return {};
+  return {std::move(fileSavePath)};
+}
```

Comment:
Functions `GetFilePathsToLoadFromGpx`, `GetFilePathsToLoadFromKml`, `GetFilePathsToLoadFromGeoJson` are very similar to me. Refactoring is needed.